apps/web/src/components/safe-messages/MsgDetails/index.tsx

```diff
@@ -23,9 +23,9 @@ import CopyButton from '@/components/common/CopyButton'
 import NamedAddressInfo from '@/components/common/NamedAddressInfo'
 import MsgShareLink from '../MsgShareLink'
 
-const MsgDetails = ({ msg }: { msg: SafeMessage }): ReactElement => {
+const MsgDetails = ({ msg }: { msg: MessageItem }): ReactElement => {
   const wallet = useWallet()
-  const isConfirmed = msg.status === SafeMessageStatus.CONFIRMED
+  const isConfirmed = msg.status === ('CONFIRMED' as const)
```

Comment:
Nit: You don't need `as const` here because the string is inherently a constant. (There are a few other `isConfirmed` intances that are the same but I won't mark them.)

Reply:
fixed.